cli/api/factory/food/food.go

```diff
@@ -0,0 +1,39 @@
+package food
+
+import (
+	"fmt"
+
+	"github.com/Seiya-Tagami/pecopeco-cli/api/model"
+	"github.com/Seiya-Tagami/pecopeco-cli/api/repository/food"
+)
+
+type factory struct {
+	repository food.Repository
+}
+
+func CreateFactory() model.FoodFactory {
+	repository := food.New()
+	return &factory{repository}
+}
+
+func (f *factory) ListFood(request food.ListRequest) ([]model.Food, error) {
+	res, err := f.repository.List(request)
+	if err != nil {
+		err := fmt.Errorf("Failed to implement Get FoodList: %v", err)
+		return []model.Food{}, err
+	}
+
+	foodList := []model.Food{}
```

Comment:
append使う時は最初にmakeしておくとパフォーマンス良くなるよ！
(再割り当てが減る)

```suggestion
	foodList := make([]model.Food, 0, len(res.Results.Shop))
```